src/renderers/noop/ReactNoop.js

```diff
@@ -146,6 +149,10 @@ var ReactNoop = {
     ReactNoop.flushLowPri();
   },
 
+  performHighPriWork(fn: Function) {
+    NoopRenderer.performWithPriority(HighPriority, fn);
```

Comment:
I agree it's too much to expose to renderers. I only added it here because I needed some way to test it. Any suggestions?